### PR TITLE
feat(context-navs): include pattern

### PR DESCRIPTION
**Pull request by kalinchernev**

### CODE REVIEW

**Comment by yhuard** [comment]
Starting the QA

File: framework/molecules/context-navs/_preview.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```
Comment (yhuard) [inline framework/molecules/context-navs/_preview.html]: Correct me if I'm wrong but you don't need this preview, the default template is enough.

Comment (kalinchernev) [inline framework/molecules/context-navs/_preview.html]: Correct! Thanks!

**Comment by yhuard** [comment]
You can merge it 👍

**Pull request merged**